types/index.d.ts

```diff
@@ -0,0 +1,35 @@
+// TypeScript Version: 3.5
+
+import { Extension, AnySchema } from '@hapi/joi'
```

Comment:
@paulmelnikow,

I had an error on import: Parsing error: 'import' and 'export' may appear only with 'sourceType: module'

Reply:
On import? Hmm, you shouldn't need to import this file.